attachments/models.py

```diff
@@ -122,7 +122,9 @@ def get_field(self, prop):
             Added for use in bootstrap's template tag render_value. Returns tuple of property label and value
         """
         if prop.data_type == 'model' and prop.slug in self.data:
-            return prop.label, prop.model_queryset().get(pk=self.data.get(prop.slug, [])[0])
+            pk = self.data.get(prop.slug, [])[0]
+            value = prop.model_queryset().get(pk=self.data.get(prop.slug, [])[0]) if pk else ''
```

Comment:
Couldn't this just be `.get(pk=pk)`?

Reply:
Also, I don't know exactly how/where this is called, but returning `None` feels more appropriate for a default than `''`. Or maybe even add a `default=None` kwarg that you could pass the empty string if you wanted.